### PR TITLE
Enable percentile output format

### LINKED ISSUE #37: Support for textual output percentile distribution
As seen on other implementation we should support outputting text percentile distributions as follows:

```
       Value     Percentile TotalCount 1/(1-Percentile)

      79.360 0.000000000000          1           1.00
      80.435 0.166666666667         17           1.20
      80.896 0.333333333333         36           1.50
      81.050 0.500000000000         52           2.00
      81.152 0.583333333333         59           2.40
      81.254 0.666666666667         70           3.00
      81.357 0.750000000000         76           4.00
      81.459 0.791666666667         86           4.80
      81.459 0.833333333333         86           6.00
      81.510 0.875000000000         93           8.00
      81.510 0.895833333333         93           9.60
      81.510 0.916666666667         93          12.00
      81.562 0.937500000000         94          16.00
      81.613 0.947916666667         98          19.20
      81.613 0.958333333333         98          24.00
      81.613 0.968750000000         98          32.00
      81.613 0.973958333333         98          38.40
      81.613 0.979166666667         98          48.00
      81.664 0.984375000000         99          64.00
      81.664 0.986979166667         99          76.80
      81.664 0.989583333333         99          96.00
      86.067 0.992187500000        100         128.00
      86.067 1.000000000000        100
#[Mean    =       80.964, StdDeviation   =        0.746]
#[Max     =       86.067, Total count    =          100]
#[Buckets =           26, SubBuckets     =         2048]
```

### PR DESCRIPTION
**Pull request by filipecosta90**
Fixes #37 
The following PR enables outputting the percentile output format, given an histogram and an io.writer. 
Included example in the documentation.

### CODE REVIEW

File: hdr.go [hunk 1]
```diff
@@ -574,3 +575,49 @@ func bitLen(x int64) (n int64) {
...
+func (h *Histogram) PercentilesPrint(writer io.Writer, ticksPerHalfDistance int32, valueScale float64) (outputWriter io.Writer, err error) {
+	outputWriter = writer
+	dist := h.CumulativeDistributionWithTicks(ticksPerHalfDistance)
+	_, err = outputWriter.Write([]byte(" Value\tPercentile\tTotalCount\t1/(1-Percentile)\n\n"))
+	if err != nil {
+		return
+	}
+	for _, slice := range dist {
+		percentile := slice.Quantile / 100.0
+		inverted_percentile := 1.0 / (1.0 - percentile)
```
Comment (ahothan) [inline hdr.go:606]: If percentile is equal to 1.0, 
java version displays "inf"
python version displays an empty string
go display "+Inf"

Maybe good to start converging.

**Review by ahothan — COMMENTED** [review]
Looks good

File: hdr.go [hunk 2]
```diff
@@ -574,3 +575,55 @@ func bitLen(x int64) (n int64) {
...
+		var inverted_percentile_string = fmt.Sprintf("%12.2f", inverted_percentile)
```
Comment (filipecosta90) [inline hdr.go:607]: @ahothan fixed per PR review. Printing in the same format as the Java version

File: hdr.go (hunk 1 above)
Comment (filipecosta90) [reply hdr.go:606]: solved the last commit

**Pull request merged**